doc/introduction.rst

```diff
@@ -158,42 +177,45 @@ a command prompt::
 Testing your installation
 .........................
 
-To test if you have done everything right, open IPython and try the
-following, in the Python prompt::
+To verify whether everything is set up correctly, open IPython (an widely
+used enhanced version of the Python prompt, cf. below) and punch in the follwing
+line::
 
     In [1]: import nilearn
 
-If you do not get any errors, you have installed nilearn correctly.
+If no errors occur, you have installed nilearn correctly.
 
 Installing the development version
 ....................................
 
-**Downloading** As an alternative to using pip, and only in the case if you
-want the latest nilearn version you can do so by using git.
+**Downloading** If you really want the latest nilearn version, you can use
+git as an alternative to using pip.
```

Comment:
"as an alternative to using pip" -> Should we put the line stating how to install the dev version with pip too?